src/plcc/load_spec/load_rough_spec/parse_dividers_test.py

```diff
@@ -50,9 +67,15 @@ def test_blocks_mask_dividers():
     lines = list(parse_blocks(parse_lines('%%%\n%\n%%%')))
     assert list(parse_dividers(lines)) == [
         Block([
-            Line('%%%', 1, None),
-            Line('%', 2, None),
-            Line('%%%', 3, None)
+            lines[0].lines[0],
+            lines[0].lines[1],
+            lines[0].lines[2]
         ])
     ]
 
+def make_divider(tool, language, line):
+    return Divider(tool=tool, language=language, line=line)
+
+def make_block(lines):
```

Comment:
This is unused. So let's delete it.

I figured out it was unused because it showed up in the coverage report.